Eatery Blue/UI/HomeScreen/HomeModelController.swift

```diff
@@ -256,13 +255,22 @@ class HomeModelController: HomeViewController {
                 }
                 currentEateries = filteredEateries
             }
-
-            eateryStartIndex = cells.count // track the index of the first eateryCard in cells
-            currentEateries.forEach { eatery in
+        }
+        let openEateries = currentEateries.filter({ $0.isOpen})
+        if (!openEateries.isEmpty) {
+            cells.append(.statusLabel(status: .open))
+            openEateries.forEach { eatery in
+                cells.append(.eateryCard(eatery: eatery))
+            }
+        }
+        let closedEateries = currentEateries.filter({ !$0.isOpen})
```

Comment:
```suggestion
        let closedEateries = currentEateries.filter { !$0.isOpen }
```

If that doesn't work, try
```swift
let closedEateries = currentEateries.filter({ !$0.isOpen })
```